Gruntfile.js

```diff
@@ -4,24 +4,62 @@ module.exports = function (grunt) {
   // Load grunt plugins
   grunt.loadNpmTasks('grunt-contrib-jshint');
   grunt.loadNpmTasks('grunt-contrib-watch');
+  grunt.loadNpmTasks('grunt-stylelint');
+  grunt.loadNpmTasks('grunt-jsonlint');
+  grunt.loadNpmTasks('grunt-htmlhint');
 
   // Plugin configuration
   grunt.initConfig({
     jshint: {
-      files: [
-        './src/**/*.js',
-        '!./src/lib/**/*'
-      ],
-      options: {
-        jshintrc: true
+      all: {
+        src: ['./**/*.js', '!./node_modules/**/*', '!./src/**/*', '!./Gruntfile.js'],
```

Comment:
It looks like with this config, you would woudn't lint anything in the src directory which is where our primary files will reside. And I like how you are considering the Gruntfile itself, but I think we _should_ lint it.

I suggest: ['./src/**/*.js', '!./src/lib/**/*', 'Gruntfile.js']